aws/modules/nlb/target/variables.tf

```diff
@@ -49,5 +49,5 @@ variable "tags" {
 variable "preserve_client_ip" {
   description = "Whether to preserve the client IP address"
   type        = bool
-  default     = false
+  nullable    = false
```

Comment:
I think we explicitly want this to be default true, I'm not even sure it's worth making this configurable TBH, but it's here so no need to remove it.